bridge_ui/src/utils/consts.ts

```diff
@@ -599,21 +606,21 @@ export const ACALA_TOKEN_BRIDGE_ADDRESS = getAddress(
 );
 export const KLAYTN_BRIDGE_ADDRESS = getAddress(
   CLUSTER === "mainnet"
-    ? "0x0000000000000000000000000000000000000000"
+    ? "0x0C21603c4f3a6387e241c0091A7EA39E43E90bb7"
```

Comment:
I am gointg to approve this, but wasn't the goal to use the addresses from the SDK, so they are only in one place?